src/components/Company/AssignSignatory/CreateSignatory/CreateSignatory.tsx

```diff
@@ -110,7 +109,7 @@ function Root({
 
       const commonData = {
         ...signatoryData,
-        birthday: birthday.toString(),
+        birthday: birthday.toISOString().split('T')[0] || '',
```

Comment:
We could use that proposed `formatForParseDate` here!

Reply:
Updated